src/theme/components/template.ts

```diff
@@ -1,8 +1,8 @@
 // From @mixin breakout:
 // Used for elements that should be edge-to-edge.
 const breakout = {
-  marginLeft: "calc(-50vw + 50%)",
-  marginRight: "calc(-50vw + 50%)",
+  marginEnd: "calc(-50vw + 50%)",
```

Comment:
I was curious if it woudl make sense to leave these as `marginLeft` and `marginRight`, but the UI appears to be working fine even with this update. :)